test/state/requests.hpp

```diff
@@ -37,9 +38,15 @@ struct Requests
     Type type = Type::deposit;
     /// Requests data - an opaque byte array, contains zero or more encoded request objects.
     evmc::bytes data;
+
+    explicit Requests(Type _type, evmc::bytes _data = {}) noexcept
```

Comment:
Needed to make `vector::emplace_back()` work for this struct on some compilers.

Reply:
I use other workaround: `emplace_back(Requests{...})`. I wonder which pattern is easier to find later...

Reply:
Usually me too, but this time I think I got error from clang-tidy "unnecessary temporary object created"